nixos/modules/system/boot/initrd-network.nix

```diff
@@ -47,6 +49,16 @@ in
       '';
     };
 
+    boot.initrd.network.udhcpc.extraArgs = mkOption {
+      default = "";
```

Comment:
Empty list maybe? Otherwise the type is not correct (also I don't know if default values are type checked).

Reply:
Good catch, thank you. Addressed in 4f0596a.